### ISSUE TITLE
hyper parameters for finetuning pegasus using official script

### ISSUE THREAD

**Issue by guangsenw** [tuner007/paraphrase#2]
Hi, first of all, thanks for the gorgeous  work. I have been using your model for some paraphrasing tasks and it performs quite well compared to others.

Now I have some other tasks and would like to finetune pegasus myself using the official training script. I am wondering whether you can share some of your finetune hyperparameters with 60K paraphrase pairs, including

1. learning rate
2. batch size (the official script suggests 256 which I cannot afford)
3. warm-up steps (official script has 10K warmup steps)
4. total num of steps

Also, regarding your use of PAWS, did you also use the "unlabelled" set in addition to the "final" and "swap" sets?
Apart from Google PAWS, do you use other dataset as well? Appreciate your help in clarification. Thanks

**Comment by monkdou0** [comment]
Same question!
I also get amazing result when using your model!
Please give more specific information about this model.
Thanks!

**Comment by guangsenw** [comment]
I think I am getting a similar performance after a week's tuning, will close the issue here. From my experience, try to use as large batch size as possible. Previously, I am using the batch_size=8 according to the official script. After digging into some closed issues in the official repo, I found that  batch_size=8 is for inference, for training, a larger batch_size (e.g., 256) should be used. For my case, I am using batch_size=32 to maximize out my GPU memory. If to use the hyper-parameters in the pegasus paper appendix, make sure the effective batch_size = 256 by increasing the learning rate and number of steps to get a satisfying performance
